test_container/build/Dockerfile

```diff
@@ -5,7 +5,9 @@ COPY deps/01_nodoc /etc/dpkg/dpkg.cfg.d/01_nodoc
 
 RUN apt-get -y update && \
     apt-get -y install \
+        openjdk-17-jdk \
```

Comment:
We probably need to set JAVA_HOME to java11 such that exaplus works. Futhermore, we should test a https server inside the UDF with Java17 to check if java17 additional jvm flags.

Reply:
We should also inform the maintainer of exaplus about the error.

EXA: import into ENGINETABLE from local csv file 'csv/ENGINETABLE.csv';



>     Error: [42636] java.sql.SQLException: ETL-5105: Following error occured while reading data from external connection [https://172.19.0.2:42375/42822b83-07e0-42bc-854e-4cbac6da4cb6 failed after 0 bytes. [OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to 172.19.0.2:42375 ],[35],[SSL connect error]] (Session: 1798057804224135168) while executing 'import into ENGINETABLE from CSV AT 'https://172.19.0.2:42375' FILE '42822b83-07e0-42bc-854e-4cbac6da4cb6' ;'; 04509 java.sql.SQLException: java.io.IOException: Exception in connect: java.lang.IllegalAccessError: class com.exasol.jdbc.importExport.KeyManager (in unnamed module @0x365f33e3) cannot access class sun.security.x509.X500Name (in module java.base) because module java.base does not export sun.security.x509 to unnamed module @0x365f33e3